pandas/plotting/_core.py

```diff
@@ -1475,7 +1475,7 @@ def pie(self, **kwargs):
         .. plot::
             :context: close-figs
 
-            >>> plot = df.plot.pie(subplots=True, figsize=(6, 3))
+            >>> plot = df.plot.pie(subplots=True, figsize=(11, 11))
```

Comment:
Like this there's lots of whitespace above and below, perhaps do this?

```suggestion
            >>> plot = df.plot.pie(subplots=True, figsize=(11, 5))
```

Reply:
yeah. I dropping a link to the image here.
![pandas-Series-plot-pie-2](https://user-images.githubusercontent.com/29908168/82576117-08573c80-9ba7-11ea-9bdf-f5082e2304dc.png)

Reply:
ok, i will change that.

Reply:
sorry i forgot add this in the description . closes #34288

Reply:
i have added the suggested changes. closes #34288

Reply:
Great, thanks!

> closes #34288

Could you add this to the pull request description? I don't know if it works as a keyword if it's just in a comment